docs/cody/capabilities/supported-models.mdx

```diff
@@ -4,6 +4,8 @@
 
 Cody supports a variety of cutting-edge large language models for use in Chat and Commands, allowing you to select the best model for your use case.
 
+<Callout type="note">Newer versions of Sourcegraph Enterprise (starting from v5.6) can be configured to use newer models as they are introduced. See [Model Configuration](/cody/clients/model-configuration) for more information.</Callout>
```

Comment:
@chrsmith 
Doesn't this statement imply that the new models will automatically be pulled? I thought this wasn't done for 5.6? 

(Also worth mentioning this is only for Cody Gateway customers, right?)

Reply:
I was kinda sorta intentionally trying to be vague here, just to avoid getting into "right now they don't" and then needing to update it later to say "oh yeah, now it does".

So "as newer models are introduced" is really "[you can update your config] as newer models are introduced... but that's a pretty weak excuse and I'll update this to be clearer.

What do you think about:

"Starting with sourcegraph vX yada yada it will be even easier to add support for new models and providers, see [Model Configuration] for more information."

Reply:
I like that rewrite 👍 Thank you 🙇